format_test.go

```diff
@@ -396,7 +396,7 @@ func TestFormatDocument(t *testing.T) {
 		string(operationDefinition.Operation),
 		operationDefinition.SelectionSet,
 	)
-	assert.Equal(t, `query {    search(id: "123") {        id        title    } }`, res)
+	assert.Equal(t, "query {\n  search(id: \"123\") {\n    id\n    title\n  }\n}", res)
```

Comment:
We could make a comparison function that compared a query ignoring whitespace? Then we don't need to tweak the tests when we change formatting?

Reply:
Yes, I agree. The whitespace in these queries are all over the place. I think this would be nice to do in another PR for now.

Reply:
Yes, I agree. The whitespace in these queries are all over the place. I think this would be nice to do in another PR for now.